src/hooks/useStockList.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { createContext, useContext, useState } from 'react';
+import { stockInfo } from '../shared/stockInfo';
+
+const StockListContext = createContext('');
```

Comment:
I am not sure why this is under hooks and not under context

Reply:
Created before merged with new code. Will fix it.